pyaedt/desktop.py

```diff
@@ -620,6 +626,14 @@ def release_desktop(self, close_projects=True, close_on_exit=True):
 
         """
         release_desktop(close_projects, close_on_exit)
+        props = [a for a in dir(self) if not a.startswith('__')]
+        for a in props:
+            try:
+                del self.__dict__[a]
+            except Exception as e:
+                pass
```

Comment:
Same as for Design.py. 
You can use the one line` self.__dict__.pop(a, None)` to remove keys if you are not sure if they exist.
Or you can use `except KeyError:` instead of `except Exception as e:`. 
Unless of course there are other reasons why it would fail.

Reply:
got it. Tks